csrc/ir/iostream.cpp

```diff
@@ -89,17 +89,24 @@ void IrTransformPrinter::handle(Fusion* f) {
 }
 
 void IrTransformPrinter::printTransforms(TensorView* tv) {
-  auto root_domain = tv->getRootDomain();
+  const auto& root_domain = tv->getRootDomain();
   os() << " root domain : (" << toDelimitedString(root_domain) << ")\n";
 
+  if (tv->hasAllocation()) {
```

Comment:
This seems fine but I wonder what the print out of a tensor should like. Should we also print the transformations between the allocation domain and the rfactor domain?

Reply:
Yeah, I was wondering how best to print it. Since allocation to rfactor will commonly not have any transforms, it's probably best to print them, so that they jump out at us. If allocation domain is just a permutation of rfactor then it will be more subtle, but the information is still there. I'll add the alloc->rfactor transforms.